src/ci/github-actions/ci.yml

```diff
@@ -293,8 +297,30 @@ jobs:
     strategy:
       matrix:
         include:
-          - name: dist-x86_64-linux
-            <<: *job-linux-xl
+          - &dist-aarch64-apple
+            name: dist-aarch64-apple
+            env:
+              SCRIPT: ./x.py dist --stage 2
+              RUST_CONFIGURE_ARGS: >-
+                --build=x86_64-apple-darwin
+                --host=aarch64-apple-darwin
+                --target=aarch64-apple-darwin
+                --enable-sanitizers
+                --enable-profiler
+                --set rust.jemalloc
+                --set llvm.ninja=false
+                --tools=rls,clippy,rustfmt,analysis,src
+              RUSTC_RETRY_LINKER_ON_SEGFAULT: 1
+              SELECT_XCODE: /Applications/Xcode_12_beta.app
+              SDKROOT: /Applications/Xcode_12_beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk
```

Comment:
This is not needed: I just checked with the DTK, and the `xcrun --sdk macosx --show-sdk-path` we're doing in `install-clang.sh` returns the correct SDK root.

Reply:
It’s needed because we aren’t running this builder on a DTK.

Reply:
That should still work. I did a test in a private repository of mine, and that command shows the 11.0 SDK path:

![Screenshot from 2020-09-21 14-08-45](https://user-images.githubusercontent.com/2299951/93764888-39f31980-fc14-11ea-9a86-03c6d782115d.png)

Reply:
Sure, I can re-enable that line of code.